config/config.json

```diff
@@ -578,20 +750,29 @@
             },
             "blockchain": {
                 "enabled": true,
-                "defaultImplementation": "otp",
+                "defaultImplementation": "otp:2043",
                 "implementation": {
-                    "otp": {
+                    "otp:2043": {
                         "enabled": true,
                         "package": "./blockchain/implementation/ot-parachain/ot-parachain-service.js",
                         "config": {
-                            "networkId": "otp::mainnet",
                             "hubContractAddress": "0x5fA7916c48Fe6D5F1738d12Ad234b78c90B4cAdA",
                             "rpcEndpoints": [
                                 "https://astrosat-parachain-rpc.origin-trail.network",
                                 "https://astrosat.origintrail.network/",
                                 "https://astrosat-2.origintrail.network/"
                             ]
                         }
+                    },
+                    "gnosis:100": {
```

Comment:
Wouldn't node try to create profile for Gnosis by default? Do we really want it to try?

Reply:
Profile will be created only if implementation is enabled